src/components/forms/JurisdictionSelect/index.tsx

```diff
@@ -90,12 +127,22 @@ export const defaultProps: Partial<JurisdictionSelectProps> = {
  * This is simply a Higher Order Component that wraps around AsyncSelect
  */
 const JurisdictionSelect = (props: JurisdictionSelectProps & FieldProps) => {
-  const { apiEndpoint, cascadingSelect, field, form, labelFieldName, params, serviceClass } = props;
-
+  const {
+    loadLocations,
+    apiEndpoint,
+    cascadingSelect,
+    field,
+    form,
+    labelFieldName,
+    params,
+    serviceClass,
+  } = props;
   const [parentId, setParentId] = useState<string>('');
   const [hierarchy, setHierarchy] = useState<SelectOption[]>([]);
   const [shouldMenuOpen, setShouldMenuOpen] = useState<boolean>(false);
   const [closeMenuOnSelect, setCloseMenuOnSelect] = useState<boolean>(false);
+  const [isJurisdiction, setIsJurisdiction] = useState<boolean>(true);
+  const [lowestLocation, setLowestLocation] = useState<boolean>(false);
```

Comment:
`lowestLocation` is never set to `true`.  What is its purpose?  Seems we may not need it

Reply:
Am passing it as a param to promiseOptions and it's being set in there as setFinalLocation(true);

Reply:
Cool, thanks for clarifying.